components/TextFields/README.md

```diff
@@ -84,10 +88,11 @@ as well as new features:
 * Labels below the input
 * Custom layouts
 * Persistable placeholder
+* Border view
 
 #### Multiline Text Field
 
-This is a multiline line text input. It's subclassed from UIView with an embedded UITextView. It supports all the features of the single line text field and UITextView plus:
+This is a multiline line text input. It's subclassed from `UIView` with an embedded `UITextView`. It supports all the features of the single line text field and `UITextView` plus:
```

Comment:
Grammar question: Is "multiline line text input" correct? "multiline text input"?

Reply:
Done.